README.md

```diff
@@ -800,3 +801,155 @@ global def pio16DUT =
 
 We should be able to reuse the test and driver for our default PIO block since
 the headers are generated based on the Object Model.
+
+## Documenting your block
+To create Scribble documentation for the new block, we will need to create three sections of text: 
+ - an "Overview" paragraph with a brief description of the block, 
+ - a "Programming" chapter describing how to program and configure the block, and
+ - a "Hardware Interface" chapter describing how to interface to the block.
+ 
+Scribble will include these sections in documents and manuals for designs which include the block.
+
+### Creating the documentation directories.
+To begin, create a directory to hold the `pio` documentation.  The directory structure allows multiple blocks to be documented,
+but in our case we are only describing `pio`.
+```
+mkdir -p docs/scribble/components/pio
+```
+In addition, edit the project's wake file to include the new scribble directory in the documentation path.
+Specifically, add the following line to `demo.wake`. 
+```wake
+publish scribbleDirectories = simplify "../../docs/scribble", Nil
+```
+
+### Creating an Onboarding document.
+At this point, we can create the first draft of the PIO Onboarding document. Type the following command.
+```
+wake makeOnboardingDocument pioDUT
+```
+The Onboarding document is a short test document which displays all the sections we've written so far. 
+It is created in the directory `build/api-generator-sifive/pioDUT/documentation`. 
```

Comment:
I think it's worth describing the file types that are produced here. Specifically, that there is an `.html` file that the user can directly view in their web browser, and that there is an `.adoc` file that shows the AsciiDoc markup.